utils/calendrical_calculations/src/rata_die.rs

```diff
@@ -47,34 +51,34 @@ impl RataDie {
         }
     }
 
-    /// A valid RataDie that is intended to be below all dates representable in calendars
-    #[doc(hidden)] // for testing only
+    /// A valid `RataDie` that is intended to be below all dates representable in calendars
+    #[doc(hidden)]
     pub const fn big_negative() -> Self {
         Self::new(i64::MIN / 256 / 256)
     }
 
-    /// Convert this to an i64 value representing the RataDie
+    /// Convert this to an `i64` value representing the `RataDie`
     pub const fn to_i64_date(self) -> i64 {
         self.0
     }
 
-    /// Convert this to an f64 value representing the RataDie
-    pub const fn to_f64_date(self) -> f64 {
+    /// Convert this to an `f64` value representing the `RataDie`
+    pub(crate) const fn to_f64_date(self) -> f64 {
         self.0 as f64
     }
 
-    /// Calculate the number of days between two RataDie in a const-friendly way
+    /// Calculate the number of days between two `RataDie` in a const-friendly way
     pub const fn const_diff(self, rhs: Self) -> i64 {
```

Comment:
Issue: I think this should be called `const_sub`, because the trait function it is shadowing is called `sub`.

Alternatively, name the function just `sub` and have it shadow. This is what we decided to do elsewhere such as `pub const fn default`.

Should it be called `sub_rata_die` since we have multiple trait impls of `sub`?